components/badge/README.md

```diff
@@ -1,51 +1,23 @@
 # Badges
 
-Badges are a lightweight method of annotating or labeling content. Wrap content in the `.badge` class to add an inline badge. The font-size and line-height of a badge inherit from that of it's parent's properties.
+Badges are a lightweight method of annotating or labeling content. Wrap content in the .badge class to add an inline badge. The font-size and line-height of a badge inherit from that of it’s parent’s properties.
 
-```
-<span class="badge">
-  Badge
-</span>
-```
+### Badge States
 
-Badge States
+You may wish to display more than the single badge type, either to create separation in the importance of various actions or to communicate the state of an action or form badge states make this super easy.
 
-You may wish to display more than the single badge type, either to create separation in the importance of various actions or to communicate the state of an action or form.  badge states make this super easy.
+|State|Purpose|Preview|
+|---	|---	|---
+|Default| |![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-default.png)|
+|Success|Represents something positive.|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-success.png)|
+|Primary|Represents something more significant than a default.|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-primary.png)|
+|Danger|Represents something in a danger or error state.|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-danger.png)|
+|Warning|Represents something that we want to warn the user about but not quite extreme as a danger state.|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-warning.png)|
+|Outline|Outline badges for when we want the density of the badge to be lighter e.g. when next to data in a table cell|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-outline.png)|
+|With Icon|If need be we can add icons and tooltips to add further context.|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-icon.png)|
```

Comment:
Do we need to mention this here? Badge is composable which means it takes every possible children component.

Reply:
Agree, but we don't want every possible child.

Reply:
Well since the badge component is a `span` this is excluding already a lot of elements and only including inline elements like `em` and `strong` for example. But then again we do not care about which children are passed to the component this is the concern of the user not of the library. And there is no sane way to check for valid components.

Reply:
Yes agree, although we want to document and have an example.

Having this will only help.